torch/ao/quantization/quantize_fx.py

```diff
@@ -134,13 +139,22 @@ def __init__(
         self.record_stack_traces = True
 
     def is_leaf_module(self, m: torch.nn.Module, module_qualified_name: str) -> bool:
+        # children of untraceable modules need to be identified so that
+        # qconfigs are not propagated to them incorrectly later.
+        if (
+            module_qualified_name in self.non_traceable_module_names
+            or type(m) in self.non_traceable_module_classes
+        ):
+            for name, child in m.named_children():
+                if not hasattr(child, "qconfig"):
+                    child.qconfig = None  # type: ignore[assignment]
```

Comment:
I feel ideally we should set qconfig in one place, to keep code easier to maintain and reason about, can we do this in prepare?